lib/internal/test_runner/reporter/spec.js

```diff
@@ -59,6 +63,36 @@ class SpecReporter extends Transform {
       ), `\n${indent}  `);
     return `\n${indent}  ${message}\n`;
   }
+  #reportCoverage(color, nesting, summary, symbol) {
+    let report = `${color}${this.#indent(nesting)}${symbol}start of coverage report${white}\n`;
```

Comment:
seems like this is duplicated from the tap reporter, can we extract the code to a common location?

Reply:
Yes 👍
where we should add this code?
I think `internal/util/` or `internal/test_runner/utils.js` one may be the right location...?

Also, can you please tell me what color and symbol we can use with the spec coverage-reporter?